### DISCUSSION TITLE
Can’t sync to PCloud WebDAV

### DISCUSSION THREAD

**Discussion by XM2008** [remotely-save/remotely-save#681] [General]
Can’t sync to PCloud WebDav

Connection test passed and local folders are correctly duplicated in workshop folder at PCloud.

When sync is tried

sync starts, then
“invalid response, 403 Forbidden”
Or “requests failed, the network connection lost”
Then “sync finished.
Any idea? Or you do have trouble connecting PCloud?
Thanks1.

**Comment by fyears** [comment]
Please don't duplicate the discussion. I am keeping the one here.

Are you using pcloud free account? Are you using the correct address based on your location?: (see, like, <https://support.enpass.io/app/sync/kb/connecting_a_vault_with_pcloud_using_webdav_in_enpass.htm> )

**Reply by XM2008** [reply]
I have paid membership.  In fact I have a few WebDAV access to my pcloud account  without any hiccups,

**Reply by fyears** [reply]
sorry but i don't have paid membership to pcloud to test...

**Comment by Juelicher-Trainee** [comment]
I am having trouble with pcloud myself. I am not well versed technically, but I noticed and veryfied one important thing:

You have to turn off 2FA (2 factor authentication) for pcloud to be able to connect via the webdav interface.
It also has to be the right webdav URL. For the European domain https://ewebdav.pcloud.com (note the "e").

Apologies if you knew all that already.

**Comment by fyears** [comment]
hi i have implemented a pcloud PRO feature with its native api instead of webdav. you are welcome to give it a try.
